tests/phpunit/tests/template.php

```diff
@@ -679,4 +724,34 @@ public function log_template_hierarchy( array $hierarchy ) {
 		return $hierarchy;
 	}
 
+	/**
+	 * Switch to premade test theme directory which contains a parent and a child theme.
+	 */
+	public function set_up_test_theme_root() {
```

Comment:
Nit-pick: This would be better as a private method since it's not needed outside the class as far as I can tell?